src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/VsServiceTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Xunit;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS
+{
+    public class VsServiceTests
+    {
+        [Fact]
+        public void Constructor_NullAsServiceProvider_ThrowsArgumentNull()
+        {
+            var threadingService = IProjectThreadingServiceFactory.Create();
+
```

Comment:
Sure, but are they really necessary in the 1-line lambda bodies? It's inconsistent with the rest of our test code.